MultiServer.py

```diff
@@ -1925,48 +1928,40 @@ def _cmd_exit(self) -> bool:
-                if name == player_name:
-                    if alias_name:
-                        alias_name = alias_name.strip()[:15]
-                        self.ctx.name_aliases[team, slot] = alias_name
-                        self.output(f"Named {player_name} as {alias_name}")
-                        update_aliases(self.ctx, team)
-                        self.ctx.save()
-                        return True
-                    else:
-                        del (self.ctx.name_aliases[team, slot])
-                        self.output(f"Removed Alias for {player_name}")
-                        update_aliases(self.ctx, team)
-                        self.ctx.save()
-                        return True
+        player = self.resolve_player(player_name)
+        if player:
+            team, slot, seeked_player = player
+            if alias_name:
+                alias_name = alias_name.strip()[:15]
+                self.ctx.name_aliases[team, slot] = alias_name
+                self.output(f"Named {seeked_player} as {alias_name}")
+                update_aliases(self.ctx, team)
+                self.ctx.save()
+                return True
+            else:
+                del (self.ctx.name_aliases[team, slot])
+                self.output(f"Removed Alias for {seeked_player}")
+                update_aliases(self.ctx, team)
+                self.ctx.save()
+                return True
         else:
-            self.output(response)
+            self.output(f"Could not find player {player_name} to alias.")
             return False
 
     def resolve_player(self, input_name: str) -> typing.Optional[typing.Tuple[int, int, str]]:
-        """ returns (team, slot, player name) """
-        # TODO: clean up once we disallow multidata < 0.3.6, which has CI unique names
-        # first match case
-        for (team, slot), name in self.ctx.player_names.items():
-            if name == input_name:
-                return team, slot, name
-
-        # if no case-sensitive match, then match without case only if there's only 1 match
-        input_lower = input_name.lower()
-        match: typing.Optional[typing.Tuple[int, int, str]] = None
-        for (team, slot), name in self.ctx.player_names.items():
-            lowered = name.lower()
-            if lowered == input_lower:
-                if match:
-                    return None  # ambiguous input_name
-                match = (team, slot, name)
-        return match
+        """
+        returns (team: int, slot: int, player name: str) if player was found or None otherwise
+        Looks for an exact match as wrong player can be fatal, _ can be used instead of a space.
+        """
+        if not self.player_name_lookup_table:
+            # populate on first use
+            for (team, slot), name in self.ctx.player_names.items():
+                self.player_name_lookup_table[name.lower().replace("_", " ")] = team, slot, name
```

Comment:
The specific segment `x.lower().replace("_", " ")` is used in enough places that I think it would be worth centralizing it as a small method.

Then, if it ever happens that it is inverted, like Black Sliver suggested above, only place place needs to be changed